src/client/pages/Debugger/components/QuoteFormNorway.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { NorwegianHomeContentsType } from 'data/graphql'
+import { InputField } from 'components/inputs'
+import { WithFormikProps } from './Offer'
+
+const initialBaseValues = {
+  firstName: 'Ole',
+  lastName: 'Olsen',
+  currentInsurer: '',
+  birthDate: '1959-11-23',
+  ssn: '23115994336',
+  startDate: '',
+  email: 'ole.olsen@hedvig.com',
+}
+
+export const initialNoHomeValues = {
+  ...initialBaseValues,
+  norwegianHomeContents: {
+    isYouth: false,
+    coInsured: 0,
+    livingSpace: 44,
+    street: 'Gulebøjsveien 1',
+    zipCode: '1234',
+    type: NorwegianHomeContentsType.Rent,
+  },
+}
+
+export const initialNoTravelValues = {
+  ...initialBaseValues,
+  norwegianTravel: {
+    coInsured: 0,
+    isYouth: false,
+  },
+}
+
+export const NorwegianHome: React.FC<WithFormikProps> = ({ formik }) => {
+  return (
+    <>
+      <InputField
+        label="Co-insured"
+        placeholder="1"
+        type="number"
+        {...formik.getFieldProps('norwegianHomeContents.coInsured')}
+      />
+      <InputField
+        label="Living space"
+        placeholder="23"
+        type="number"
+        {...formik.getFieldProps('norwegianHomeContents.livingSpace')}
+      />
+      <InputField
+        label="Street"
+        placeholder="Gulebøjsveien 1"
+        {...formik.getFieldProps('norwegianHomeContents.street')}
+      />
+      <InputField
+        label="Zip code"
+        placeholder="1234"
+        {...formik.getFieldProps('norwegianHomeContents.zipCode')}
+      />
+      <InputField
+        label="Type"
+        placeholder=""
```

Comment:
same do we need it?

Reply:
Yes :)
![image](https://user-images.githubusercontent.com/42962286/97291588-6a008e80-184a-11eb-9539-19b98b97c3b2.png)

Reply:
But we actually don't need `ssn` so maybe I should remove that